database/seedScript.js

```diff
@@ -12,7 +12,7 @@ const seed = (numberSeeds) => {
 
   Reviews.create(entriesList)
     .then(() => connection.close())
-    .catch((err) => console.error(err));
+    .catch((err) => console.error(err)); //eslint-disable-line
```

Comment:
I think the "right" way to do this is throw an error instead of using the console. You can also import the error function from the console library and use that, it's the same thing you're doing here but eslint won't bug you about it.